src/Command/PqCommands/ContentHubPqAsymmetricParagraphs.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace Acquia\Console\ContentHub\Command\PqCommands;
+
+use Acquia\Console\ContentHub\Command\Helpers\DrupalServiceFactory;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * Runs check against paragraphs which could be asymmetric and lists them down.
+ */
+class ContentHubPqAsymmetricParagraphs extends ContentHubPqCommandBase {
+
+  /**
+   * Paragraph storage.
+   */
+  const PARAGRAPH_STORAGE = 'paragraph';
+
+  /**
+   * The Drupal service factory service.
+   *
+   * @var \Acquia\Console\ContentHub\Command\Helpers\DrupalServiceFactory
+   */
+  protected $serviceFactory;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $defaultName = 'ach:pq:asymmetric-paragraphs';
+
+  /**
+   * Constructs a new ContentHubPqEntityTypes object.
+   *
+   * @param \Acquia\Console\ContentHub\Command\Helpers\DrupalServiceFactory $drupalServiceFactory
+   *   The Drupal service factory service.
+   * @param string|null $name
+   *   The name of this command.
+   */
+  public function __construct(DrupalServiceFactory $drupalServiceFactory, string $name = NULL) {
+    parent::__construct($name);
+    $this->serviceFactory = $drupalServiceFactory;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function configure() {
+    parent::configure();
+    $this->setDescription('Analyses paragraphs and checks whether they are asymmetric based on site\'s default language.');
+    $this->setAliases(['ach-pq-ap']);
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function runCommand(InputInterface $input, PqCommandResult $result): int {
+    /** @var \Drupal\Core\Entity\EntityTypeManagerInterface $etm */
+    $etm = $this->serviceFactory->getDrupalService('entity_type.manager');
+    $bundles = $this->getBundles(self::PARAGRAPH_STORAGE, $etm);
```

Comment:
if this is an empty array we can return here earlier as well.

Reply:
Done.